### ISSUE TITLE
Error when copying `.../share/man`

### ISSUE THREAD

**Issue by hangtwenty** [tj/n#100]
Using Manjaro Linux 0.83 (XFCE Edition). Installed node 0.8.20 manually then used npm to install n. Have since used n to install several versions of node. Everything seems to be working, including switching node versions (as long as I do `sudo` but I figure that's given). I do get this error however:

```
cp: cannot overwrite non-directory ‘/usr/local/share/man’ with directory ‘/usr/local/n/versions/0.9.9/share/man’
```

Indeed, this is because `/usr/local/share/man` is a symbolic link to `/usr/local/man`. Not sure if this is applicable to Arch in general or just Manjaro.

Again, things still seem to be working; all that seems to be faltering is the copying of the man page, and that's fine really. Nonetheless thought it might be worth reporting.

**Comment by qw3rtman** [comment]
Is this still an issue?

**Comment by ntrrgc** [comment]
@qw3rtman Still happening in 2.1.3

**Comment by qw3rtman** [comment]
@ntrrgc, what OS are you running?

**Comment by ntrrgc** [comment]
Arch Linux.

**Comment by deepsweet** [comment]
same error here with Arch Linux.

**Comment by pcstl** [comment]
Same error happening on Antergos Linux. I'm copying manpages manually from ```/usr/local/n/versions/node/{version}/share/man``` to ```/usr/share/man``` as a workaround.

**Comment by winneon** [comment]
Still happening. Has there been any progress on this?

**Comment by rku4er** [comment]
I have noticed it works well if I run with sudo (Arch Linux)

**Comment by devzsolt** [comment]
Still broken on Manjaro Linux. Any fix perhaps?

**Comment by JohnRGee** [comment]
A manual work-around might be to install to a different location using `N_PREFIX`, and add to `PATH` or add symbolic links in `/usr/local`. Likewise, there is third party installer [n-install](
https://github.com/mklement0/n-install).

(A little history. The suggestion to fix this issue by using `rsync` instead of `cp` got rejected in #104. Support got added later to cope with symbolic links as direct subdirectories of `/usr/local` such as `/usr/local/bin` in #227, but not deeper in the hierarchy such as `/usr/local/share/man`.)

**Comment by devzsolt** [comment]
Thank you @JohnRGee the recommended `n-install` solution works like a charm on the latest Manjaro, so possibly on other Arch-based distros too. I also tried installing n from an AUR package called `nodejs-n` but despite it was prepared for Arch systems it had this same issue with the symlink dir. So n-install is the way to go for now.

**Comment by shadowspawn** [comment]
If readers want to install to `/usr/local` and a fork of `n` is an option, I have switched to rsync in [nvh](https://github.com/shadowspawn/nvh) which avoids this symlink problem.

**Comment by hangtwenty** [comment]
Sorry if other users disagree but since I opened this in 2013 and have not touched arch linux since that year or year after, I'm closing this issue. Arch Linux was a great way to force myself to "live" in the CLI (and I still do today) but even if I went back, I would not expect nonessential conveniences like `n` to work there. If someone else feels differently they can open another ticket and link this one or something, just casting my "vote" to close it... since I was the OP.
